### PR TITLE
Updated a few elements in boilerplate .adoc files

### PR DESCRIPTION
**Pull request by MarciaRieferJohnston**
Changed header style from level 3 to bold so that this header stops triggering a "section title out of sequence" warning when we generate the index.html file during doc review.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by censullo — APPROVED** [review]

**Comment by jaymccon** [comment]
The merge conflicts are due my changing the base branch to main, will try to get those resolved shortly.

**Review by jaymccon — APPROVED** [review]

File: .specific/additional_info.adoc [hunk 1]
```diff
@@ -4,8 +4,8 @@
 == Test the deployment
 // If steps are required to test the deployment, add them here. If not, remove the heading
 
-== Post deployment steps
-// If Post-deployment steps are required, add them here. If not, remove the heading
+== Postdeployment steps
```
Comment (andrew-glenn) [inline .specific/additional_info.adoc:7]: "Post deployment" / "Post-deployment" is an industry accepted term. Please do not deviate.

**Review by andrew-glenn — CHANGES_REQUESTED** [review]

File: disclaimer.adoc [hunk 2]
```diff
@@ -1,6 +1,6 @@
-© 2020, Amazon Web Services Inc., or its affiliates, and {partner-company-name}. All rights reserved.
+© 2020 Amazon Web Services, Inc., or its affiliates and {partner-company-name}. All rights reserved.
 
-==== Notices
```
Comment (andrew-glenn) [inline disclaimer.adoc:3]: Please retain this indentation level.

**Review by andrew-glenn — CHANGES_REQUESTED** [review]

File: .specific/additional_info.adoc (hunk 1 above)
Comment (MarciaRieferJohnston) [reply .specific/additional_info.adoc:7]: @andrew-glenn Thanks for this feedback. The tech writers can see aligning on "post-deployment" (despite the no-hyphen guideline) after reviewing AWS usage elsewhere. The current boilerplate heading is "Post deployment steps." I'll replace the space with a hyphen.

**Comment by andrew-glenn** [comment]
_Discussed with Marcia out-of-band. This PR is in a weird state with no clear path forward, due to an unknown base. Asking her to manually pluck any relevant changes into a new PR. Closing._

**Pull request closed without merging**